### ISSUE TITLE
Invalid Decrption Key!?

### ISSUE THREAD

**Issue by marshy2** [linzhouyu/ACST#1]
I used your program once, and it worked.  But I had to update my save files & now it's not working & instead just showing me this.

![image](https://user-images.githubusercontent.com/3310842/48965241-8e74d180-ef76-11e8-938b-c5fb4cefc841.png)

PLEASE HELP.

Write instructions with pictures please!

**Comment by linzhouyu** [comment]
Could you specify how you "update your saves"? Also, please try detect key function to see if it works.

**Comment by marshy2** [comment]
@linzhouyu Sorry man, I eventually got it working, just forgot to update it here.  The problem was that I was using the .bak files it created and renaming them to .save instead of using the .save files.  I found your instructions on this website https://m.jb51.net/game/646045.html
and I just had to read them more carefully.  I think it would be great if you included the instructions in the readme file for the future.  Thanks so much for the help!
